macros/get_base_dates.sql

```diff
@@ -1,4 +1,8 @@
 {% macro get_base_dates(start_date=None, end_date=None, n_dateparts=None, datepart=None) %}
+    {{ adapter.dispatch('get_base_dates', packages = dbt_date._get_utils_namespaces()) (start_date, end_date) }}
```

Comment:
Hi @VasiliiSurov, love the adapter macro version here, but I think you need to also pass the additional parameters like `n_dateparts` and `datepart` to the call to `dispatch` and also the subsequent macros it's calling, right?